actions/dryRunInstruction.ts

```diff
@@ -15,6 +16,11 @@ export async function dryRunInstruction(
   additionalInstructions?: InstructionData[]
 ) {
   const transaction = new Transaction({ feePayer: wallet.publicKey })
+
+  transaction.add(
```

Comment:
Is this desireable? Might it cause transactions to succeed in dry run that would fail in the real world? The answer may be no, I have no idea.

Reply:
compute unit is already bumped to 1_000_000 in both executeInstructions.ts and executeTransaction.ts. There might be a better way of doing this but if we didn't have it here thinks would fail which would otherwise pass in both of those.

Reply:
Got it. sounds good.